tools/accuracy_checker/accuracy_checker/preprocessor/geometric_transformations.py

```diff
@@ -380,6 +381,9 @@ def parameters(cls):
             ),
             'use_numpy': BoolField(
                 optional=True, default=False, description="Allow to use numpy for padding instead default OpenCV."
+            ),
+            'numpy_pad_mode': StringField(
+                optional=True, default='constant', description="If use_numpy is True, Numpy padding mode,including constant, edge, mean, etc."
```

Comment:
it makes sense to have the list of possible values for this parameter.
please specify it with `choices`  argument